src/test/java/io/vertx/test/core/TimerTest.java

```diff
@@ -163,26 +164,33 @@ public void testTimerStreamExceptionDuringHandle() throws Exception {
   @Test
   public void testTimerStreamCallingWithNullHandlerCancelsTheTimer() throws Exception {
     ReadStream<Long> timer = vertx.timerStream(10);
+    AtomicInteger count = new AtomicInteger();
     timer.handler(l -> {
-      fail();
-    });
-    timer.endHandler(v -> {
-      testComplete();
+      if (count.incrementAndGet() == 1) {
+        timer.handler(null);
+        vertx.setTimer(200, id -> {
+          assertEquals(1, count.get());
```

Comment:
If you want to verify a timer is only called once, then probably need to wait a bit for completing test